examples/standalone_mounted_disk/versions.tf

```
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.79"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.0.0"
```

Just to keep things more flexible:
```suggestion
      version = "~> 3.0"
```